packages/material/scss/grid/_layout.scss

```diff
@@ -0,0 +1,178 @@
+@import "~@progress/kendo-theme-default/scss/grid/layout";
+
+@include exports ('grid/layout/material') {
+    $grid-header-font-size: 12px !default;
+    $grid-filter-button-offset: -10px !default;
+    $grid-filter-button-size: 36px !default;
+    $grid-header-line-height: 16px !default;
+    $grid-header-cell-padding-y: 20px !default;
+    $grid-cell-padding-x: 24px !default;
+    $grid-cell-padding-y: 10px !default;
+    $grid-header-padding-right: 14px !default;
+    $grid-hierarchy-col-width: 8px !default;
+    $grid-grouping-header-padding-y: 16px !default;
+    $grid-group-indicator-margin: 8px !default;
+    $grid-grouping-indicator-border-radius: calc( #{$font-size} + #{$button-padding-y}) !default;
+    $grouping-dropclue-height: calc( ( 2 * #{$font-size}) + (2 * #{$padding-y})) !default;
+    $grouping-dropclue-margin-top: calc( #{$grid-header-padding-right} - 1px) !default;
+    $grid-filter-row-padding-y: 13px !default;
+
+    .k-grid {
+        th {
+            border-bottom-width: 1px;
+        }
+
+        th.k-header {
+            font-size: $grid-header-font-size;
+            line-height: $grid-header-line-height;
+        }
+
+        .k-grid-header-wrap th.k-header,
+        .k-grid-header-locked th.k-header {
+            padding: $grid-header-cell-padding-y $grid-header-padding-right $grid-header-cell-padding-y $grid-cell-padding-x;
```

Comment:
Having padding-x, padding-y and padding-right seems suspicious. Perhaps naming the last one to something more specific will improve this code.